swap/tests/alice_manually_punishes_after_bob_dead_and_bob_cancels.rs

```diff
@@ -0,0 +1,83 @@
+pub mod harness;
```

Comment:
This is a reasonable test but is it required for this PR? 

Honest question. Are you testing any behaviour here that was changed by your changes?